package.json

```diff
@@ -25,6 +25,9 @@
     "docs:copy-index": "gulp docs-copy-index",
     "docs:sitemap": "gulp create-sitemap",
     "docs:gh-pages": "ts-node -P tools/deploy-docs/tsconfig.json tools/deploy-docs/build-docs.ts",
+    "docs:create-routes": "gulp create-docs-routes",
+    "docs:prerender": "ng run docs:prerender --routes-file ./docs/routes.txt",
+    "docs:build-prerender": "npm-run-all docs:prepare docs:create-routes docs:prerender",
```

Comment:
Keep the main task here and move other tasks to the `predocs:build-prerender` script

Reply:
done